README.md

```diff
@@ -0,0 +1,17 @@
+#### Our Project
+
+Web based postal client
+
+#### Current features
+
+- None yet
+- But we have some uml-s and web-gui design ideas!
```

Comment:
I think we have some features for now, so it could be complited

Reply:
yeah, we definitely need to update this file with our new features and set up description for uml-s in more appropriate way

Reply:
yeah, we definitely need to update this file with our new features and set up description for uml-s in more appropriate way